portal/app/api/v1/endpoint/farm_device.py

```diff
@@ -20,4 +26,12 @@ def get_queryset(self, **kwargs):
         return queryset
 
 
+class FarmDataViewSet(viewsets.ModelViewSet):
+    authentication_classes = (CsrfExemptAuth.CsrfExemptSessionAuthentication, TokenAuthentication)
+    serializer_class = FarmDataSerializer
+
+    def get_queryset(self, **kwargs):
+        queryset = FarmData.objects.filter(home__owner__user=self.request.user).order_by('-id')
```

Comment:
FarmData doesn't have a home property, so this filter won't work

Reply:
That's what I figured. Working on this...